api/controllers/MerchStoreController.ts

```diff
@@ -188,4 +197,16 @@ export class MerchStoreController {
     await this.merchStoreService.updateOrderItems(fulfillOrderRequest.items);
     return { error: null };
   }
+
+  private verifyMerchOrderRequest(orderRequest:MerchItemOptionAndQuantity[],
+    user:UserModel):MerchItemOptionAndQuantity[] {
+    if (!PermissionsService.canAccessMerchStore(user)) throw new ForbiddenError();
```

Comment:
Can you move this permissions check out of this method and back to the endpoints? This method should only validate the request.